### ISSUE TITLE
sqlauth module does not work with sqlite3 ("database is locked")

### ISSUE THREAD

**Issue by bitc** [inspircd/inspircd#1453]
I am trying to use the "m_sqlauth" module with "m_sqlite3" but I keep getting the error message "SQL query failed: database is locked":

```
Tue Jan  2 16:38:08 2018: ANNOUNCEMENT: Forbidden connection from bitc!bitc@203.0.113.0 (SQL query failed: database is locked)
```

**Output of `./bin/inspircd --version`:** InspIRCd-2.0.24 r0

When I use MySQL everything works great, but sqlite is giving me the above error.

I am sure that the sqlite file I created is correct and inspircd seems to be connecting to it ok. (sqlite database created with sqlite version 3.19.3)

When I set the sqlauth "query" to `SELECT 'ok'` then the auth succeeds and the user is able to connect.

When I set the "query" to `SELECT sqlite_version()` then it also works, which tells me that inspircd is indeed properly connected to sqlite. (`SELECT bla_blah()` does not work).

But as soon as I try to access a table, for example, using the query `SELECT username FROM my_users`, then I get the above error "database is locked".

I checked, and no other process on the system is accessing the sqlite database file.

Thank you

**Comment by SISheogorath** [comment]
Sounds more like a config problem than a real bug. And this way is misplaced on a bug tracker. Please visit #inspircd on irc.inspircd.org for this kind of requests.

**Comment by bitc** [comment]
This seems to be a problem with Ubuntu 17.10 inspircd package.

When I compile inspircd from source it works great 👍 
I compiled inspircd git master, and inspircd git tag 2.0.24 (the version that ubuntu uses) and both work.

I looked at the ubuntu package to see what patches they applied, but none of them look like they would have any affect on this.

So still not sure what's going on. But I guess this issue can be closed
